internal/clients/nightfall/nightfall.go

```diff
@@ -47,7 +47,7 @@ var (
 type Client struct {
 	APIClient          nightfallintf.NightfallAPI
 	APIKey             string
-	Detectors          []*nightfallAPI.Detector
+	Conditions         []*nightfallAPI.Condition
```

Comment:
either create a struct containing both `[]*nightfallAPI.Condition` and `UUID`, or simply add a `ConditionSetUUID` field in the case where users just input a pre-built condition set uuid